### ISSUE TITLE
Private chat test case.

### ISSUE THREAD

**Issue by sakky27** [bigbluebutton/bigbluebutton#13957]
# Describe the bug
Users can see the private chat they had with other user even though private chat lock is enabled. 

# To Reproduce
1) As a viewer send a private chat to another viewer.
2) As a moderator go to manage users, select lock viewers and enable private chat messages locked.
3) Viewer should try to access the previously private chat with other user.

# Expected behavior
Once private chat lock is enabled, if there was an existing open private chat tab between 2 users, they should be unavailable to viewers when the lock is enabled.

# Actual behavior
Users should not be able to send as well as see the private chats as per the test case.

# Screenshots

1.  Moderator has enabled the private chat locked.

![image](https://user-images.githubusercontent.com/43130870/146837999-38e36c9a-26d7-4c8c-9da5-99c64645c520.png)

2.  Both users, user1 and user3 can see the previously private chat even though private chat lock is enabled.

![Capture2](https://user-images.githubusercontent.com/43130870/146838054-f59331d1-525d-45b0-a5c6-11681377eb6c.PNG)

![Capture3](https://user-images.githubusercontent.com/43130870/146838038-0ee7d3e4-1a40-416c-b56f-19bd7e7be34e.PNG)

# BBB version:

BigBlueButton Server 2.4

# Desktop:
 - OS: [e.g. Windows
 - Browser [e.g. Chrome, Edge]
 - Version [e.g. 24]

# Additional context
Test case as per the documentation is failing.

**Comment by antobinary** [comment]
I tried reproducing this on 2.4.2

Lock settings:
![Screenshot from 2022-01-19 12-37-15](https://user-images.githubusercontent.com/6312397/150184455-b0fccea3-9516-4170-8ded-304353e4c92a.png)

When I tried to initiate chat viewer -> viewer, I could not open the menu
![Screenshot from 2022-01-19 12-37-28](https://user-images.githubusercontent.com/6312397/150184532-ac5612cf-7d8b-46a4-be41-09013b639550.png)

When I tried to initiate chat viewer -> moderator, that worked, but it's a special case - by design we want to allow students to signal to the instructor at any time
![Screenshot from 2022-01-19 12-37-35](https://user-images.githubusercontent.com/6312397/150184529-3189d750-cffc-4ca6-a2a6-ec37fc0c2dfa.png)


![Screenshot from 2022-01-19 12-40-50](https://user-images.githubusercontent.com/6312397/150184909-8e423166-1e12-40e2-a797-3ebdac69b83f.png)
If there was an existing private chat thread before the lock setting was enforced, it is unusable once the private chat feature is locked.


tl;dr: I am unable to reproduce this issue on BBB 2.4.2
@sakky27 could you please try it again and possibly record video if reproducible - there must be some extra step I'm missing

**Comment by antobinary** [comment]
I followed up with @sakky27 and seems the key part was whether old private chat history can be viewed or not.
We currently allow viewing old chat history during private chat lock.

If moderators want to prevent private chat exchange, the lock setting should be set up and enforced on meeting create.

Thanks @sakky27 for reporting this but I think we'll keep the behaviour currently implmented
